Dockerfile

```diff
@@ -13,13 +13,13 @@ RUN apt-get -y update \
   && update-alternatives --install /usr/bin/python python /usr/bin/python3.8 1 \
   && python -m pip install --upgrade pip \
   && python -m pip install --upgrade setuptools \
-  && apt-get clean
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
I've tried only this line change with docker build --no-cache command and this does not reduce the size of the image, but this brokes possibility to install new packages with apt-get command in interactive mode.
I suggest removing this from the PR.

Reply:
@ometelytsia if you will use apt in interactive mode after a few days (or month) since container build - apt cache will be outdated. So, in case if you would like to use apt - you should start with 'apt update' anyway.

Reply:
What is the benefit of this change if the image size stays the same? The size of this folder is 29Mb. Also, we do have a CI system that builds `dev` version of this container on a regular basis from the same Dockerfile and a couple of test containers that are based on the `atlassian/dcapt` container. So I do not see any benefits from this change, rather than additional troubles for our development pipeline.

Reply:
@ometelytsia I don't see any potential issues in development process, but I don't want to force push any changes. So, I'm ok with keeping this 29 Mb, knowing about 3.8Gb of full container size :)